dataprep/clean/clean_headers.py

```diff
@@ -0,0 +1,246 @@
+"""
+Implement clean_headers function
```

Comment:
Can you please take a look at point 1. (Module Docstring) [here](https://github.com/sfu-db/dataprep/wiki/Docstring-Best-Practice#dataprepclean-docstring-recommendations). I'm wondering if we could write a clearer brief description of the functionality in this module, perhaps something like "Clean and standardize column headers." (or whatever you think is best)